orbyter-dl-dev/README.md

```diff
@@ -18,16 +18,57 @@ Now using `orbyter-base-sys:2.0` as a base image.
 
 #### Removed
 
-cufflinks==0.16
-
 #### Updated
 
-streamlit==0.50.1
-notebook==6.0.2
+* bayesian-optimization==1.2.0
```

Comment:
Here, I would just list out what you updated, not the entire `pip freeze`

Reply:
Oh wow. Maybe it is actually all these things?

Reply:
Are these the same as `ml-dev:2.1`?

Reply:
Yeah I ran `pip list --outdated` and found many packages that were not up to date. Updating the packages would make this version closer to `mldev:3.2` instead of `mldev:2.1`. Should I bump the version to become 3.2 then? Is the idea that the `dldev` image version should closely mirror `mldev` version + additional DL libraries?